ImperatorToCK3/Source/CK3Outputter/outTitles.cpp

```diff
@@ -17,6 +17,9 @@ void CK3::outputTitles(const std::string& outputModName, const std::string& ck3P
 	
 	for (const auto& [name, title] : titles)
 	{
+		if (title->imperatorCountry.second && title->imperatorCountry.second->getCountryType() != "real") // we don't need pirates, barbarians etc.
```

Comment:
Is it possible to use a stronger type for countryType? Or do we ultimately care about more than real/unreal distinction. If not, try using a bool instead.

Reply:
That caught my eye too but then I realized it was a literal string from the save. That's how they mark "real" countries, hah.

Reply:
I want to keep it as string or enum class, because "barbarian" or sth type can also occupy provinces in Imperator AFAIK.

Reply:
Yeah, I had the same double-take. I'm inclined to ask for an enum, but its understandable of that seems a bit heavy for the purpose.